lib/rsk-utils.js

```diff
@@ -559,6 +567,33 @@ const waitForRskMempoolToGetAtLeastThisManyTxs = async (rskTxHelper, atLeastExpe
   throw new Error(`[${waitForRskMempoolToGetAtLeastThisManyTxs.name}] rsk mempool didn't get at least ${atLeastExpectedCount} txs after ${maxAttempts} attempts`);
 };
 
+const uncompressPublicKey = (compressedPublicKey) => {
+  const uncompressedPublicKey = secp.ProjectivePoint.fromHex(removePrefix0x(compressedPublicKey));
+  return uncompressedPublicKey.toHex(false);
+};
+
+const keccak256 = (str) => {
+  return Web3.utils.keccak256(str);
+}
+
+const removeCompressionPrefix = (uncompressedPublicKey) => {
+  if(uncompressedPublicKey.startsWith('04')) {
+      return uncompressedPublicKey.substring(2);
+  } else if(uncompressedPublicKey.startsWith('0x04')) {
+      return uncompressedPublicKey.substring(4);
+  }
```

Comment:
a nit one: 

```suggestion
  if (uncompressedPublicKey.startsWith('0x04')) {
      return uncompressedPublicKey.substring(4);
  }
  if (uncompressedPublicKey.startsWith('04')) {
      return uncompressedPublicKey.substring(2);
  } 
```

i find this way more straightforward

Reply:
I agree with this one.